src/client/getConsumerClientFactory.js

```diff
@@ -0,0 +1,22 @@
+export function getConsumerClientFactory() {
+  const factories = {};
+
+  // todo: can we have both clients on node.js ?
```

Comment:
If we will be implementing this in the future it's worth mentioning that websockets are not working properly inside Node.js out of the box for some reason. So websocket client for Node.js won't be only about client creation